docs/usage-examples/deleteOne.txt

```diff
@@ -0,0 +1,66 @@
+.. _laravel-delete-one-usage:
+
+=================
+Delete a Document
+=================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: delete one, remove, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can delete a document in a collection by retrieving a single document and calling
+the ``delete()`` method on an Eloquent model or a query builder.
+
+Pass a query filter to the ``where()`` method, sort the matching documents, and call the
+``first()`` method to retrieve only the first document. Then, delete this matching document
+by calling the ``delete()`` method.
```

Comment:
I think this paragraph could be more clear if it was presented as a list: 

```suggestion
Perform the following steps to delete a document:

1. Pass a query filter to the ``where()`` method
2.  Sort the matching documents
3.  Call the ``first()`` method to retrieve only the first document 
4.  Delete this matching document by calling the ``delete()`` method.
```

Although, looking at the update usage example page, it looks like that might not be inline with the formatting being used in these docs. An alternative suggestion would be:

```suggestion
To delete a document, pass a query filter to the ``where()`` method, sort the matching documents, and call the
``first()`` method to retrieve only the first document. Then, delete this matching document
by calling the ``delete()`` method.
```

Reply:
Took your second suggestion!